src/common/style/vanilla/button.scss

```diff
@@ -0,0 +1,4 @@
+@import 'vanilla-framework/scss/base_button.scss';
```

Comment:
`pattern_buttons` requires a mixin from`base_button`.

In theory importing whole vanilla should solve it (include all the mixins), but because of #1137 it includes some unnecessary CSS.